src/app/app.js

```diff
@@ -64,19 +64,22 @@ document.addEventListener('DOMContentLoaded', () => {
   // all sections's buttons toggle
   const sections = document.querySelectorAll('.js--section');
 
-  Array.from(sections).forEach((section) => {
-    let lastEnabled = null;
-    const toggleButtons = section.querySelectorAll('.js--toggle');
-    const arr = Array.from(toggleButtons);
-    [lastEnabled] = arr;
-    arr.forEach((button) => {
-      button.addEventListener('click', () => {
-        disable(lastEnabled, section);
-        enable(button, section);
-        lastEnabled = button;
+  // let the components render first, thats why this hack
```

Comment:
what about using event delegation here instead? by `on` module

Reply:
i think is ok with settimeout as its not critical code. Its just executed before the Components cause their DOM Ready listener is later in the queue. I can comment it better thought so is clear why